examples/ex_nsga2.cpp

```diff
@@ -65,10 +81,19 @@ SFERES_FITNESS(FitZDT2, sferes::fit::Fitness) {
 int main(int argc, char **argv) {
   std::cout<<"running "<<argv[0]<<" ... try --help for options (verbose)"<<std::endl;
 
+#ifndef NO_PARALLEL
+#ifndef NO_MPI
+  typedef eval::Mpi<Params> eval_t;
```

Comment:
I think we should not default to MPI. The default should be eval::Parallel and eval::Eval otherwise.